contrib/installer/linux/voxedit.desktop.in

```diff
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Version=1.0
-Name=@NAME@ @CMAKE_PROJECT_VERSION@
-GenericName=@NAME@
+Name=@NAME@
```

Comment:
Any particular reason the version was removed here?

Reply:
Yes, it does not belong there.